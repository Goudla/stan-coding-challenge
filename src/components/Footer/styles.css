.footer {
  background-color: #1e1e1e;
  bottom: 0;
  min-height: 215px;
  position: absolute;
  width: 100%;
}
