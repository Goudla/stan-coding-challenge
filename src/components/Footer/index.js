import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class Footer extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="footer">
        {children}
      </div>
    );
  }
}

export default Footer;
