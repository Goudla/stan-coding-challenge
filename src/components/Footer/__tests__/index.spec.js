import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import Footer from '../';

describe('<Footer />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Footer>Test</Footer>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
