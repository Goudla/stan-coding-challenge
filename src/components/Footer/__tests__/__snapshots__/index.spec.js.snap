// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should render 1`] = `
<div
  className="footer"
>
  Test
</div>
`;
